setup.cfg

```diff
@@ -60,4 +60,4 @@ max-attributes = 11
 ignore-comments = yes
 ignore-docstrings = yes
 ignore-imports = yes
-disable = R0903, R0913, R0914, C0209, W0223, W0221, W0237
+disable = R0801 R0903, R0913, R0914, C0209, W0223, W0221, W0237
```

Comment:
What is this for?

Reply:
This disables the warning about duplicate code that PyLint raises because the  `get_output_types` functions are very similar across multiple transformers.
This is temporary and will be removed in a near-future update when the NullTransformer is decoupled as an independent thing.
@fealho The syntax is wrong, though, because the warnings need to be separated by comma (look at the others) and when the comma is missing PyLint does not take it. I'm correcting the syntax myself now